backend/applications/models.py

```diff
@@ -0,0 +1,105 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from django_cryptography.fields import encrypt
+
+from applications.enums import ApplicationStatus
+from companies.models import Company
+from utils.models import HistoricalModel, UUIDModel
+
+
+class Application(HistoricalModel, UUIDModel):
+    company = models.ForeignKey(
+        Company,
+        on_delete=models.CASCADE,
+        related_name="applications",
+        verbose_name=_("company"),
+    )
+    status = models.CharField(
+        max_length=64, verbose_name=_("status"), choices=ApplicationStatus.choices
+    )
+
+    invoicer_name = models.CharField(
+        blank=True,
+        null=True,
+        max_length=256,
+        verbose_name=_("invoicer name"),
+    )
+    invoicer_email = models.EmailField(
+        blank=True, null=True, verbose_name=_("invoicer email")
+    )
+    invoicer_phone_number = models.CharField(
+        max_length=64,
+        blank=True,
+        null=True,
+        verbose_name=_("invoicer phone number"),
+    )
+
+
+class SummerBill(HistoricalModel, UUIDModel):
+    application = models.ForeignKey(
+        Application,
+        on_delete=models.CASCADE,
+        related_name="summer_bills",
+        verbose_name=_("application"),
+    )
+    summer_bill_id = models.CharField(
+        max_length=256, blank=True, null=True, verbose_name=_("summer bill id")
```

Comment:
Same here, avoid using `null=True` on text-based fields. This applies to these fields:

- summer_bill_id
- contact_name
- work_postcode
- employee_name
- employee_school
- employee_ssn
- employee_phone_number